bin/get_tokens

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require "base64"
+require 'dotenv'
+require 'httparty'
+
+Dotenv.load('.env.test')
+
+print "Let's get you an access token and a refresh token for Fortnox API."
+
+client_id = ENV.fetch('FORTNOX_API_CLIENT_ID')
+client_secret = ENV.fetch('FORTNOX_API_CLIENT_SECRET')
+redirect_uri = ENV.fetch('FORTNOX_API_REDIRECT_URI')
+scopes = ENV.fetch('FORTNOX_API_SCOPES')
+credentials = "#{Base64.encode64("#{client_id}:#{client_secret}")}"
+oauth_endpoint = 'https://apps.fortnox.se/oauth-v1'
```

Comment:
Constant?

Reply:
Do you mean "Constant?" as in "is this is always the endpoint"? In that case - yes, this is the endpoint Fortnox uses according to https://developer.fortnox.se/general/authentication/.